eng/pipelines/templates/jobs/ci.yml

```diff
@@ -398,6 +398,12 @@ jobs:
         parameters:
           JdkVersion: ${{ parameters.JavaBuildVersion }}
 
+      - task: NodeTool@0
+        condition: and(succeededOrFailed(), ne(variables['Skip.SpellCheck'],'true'))
```

Comment:
Curious why we need this condition.

Reply:
It's a mistake when I copied from common (the one before cspell)...